### DISCUSSION TITLE
blog/2026/03/11/the-secret-low-rank-bias-of-regularized-sgd

### DISCUSSION THREAD

**Discussion by giscus** [DLFundamentals/blog#3] [Announcements]
# blog/2026/03/11/the-secret-low-rank-bias-of-regularized-sgd

Why do trained neural networks often end up low rank? Mini-batch SGD and weight decay together create a built-in pressure toward compressible layers.

https://dlfundamentals.github.io/blog/2026/03/11/the-secret-low-rank-bias-of-regularized-sgd.html
